tools/accuracy_checker/accuracy_checker/metrics/reid.py

```diff
@@ -231,31 +231,64 @@ def parameters(cls):
                 description="Min score for determining that objects are different. "
                             "You can provide value or use train_median value which will be calculated "
                             "if annotations has training subset."
+            ),
+            'distance_mode': StringField(
+                optional=True, default='0', description='Allows to choose one of the distance calculation mode',
+                choices=['0', '1']
```

Comment:
I think better to use real methods names instead euclidian_distance or cosine distance

Reply:
Done